src/core/client_channel/client_channel.cc

```diff
@@ -1256,6 +1259,9 @@ void ClientChannel::StartIdleTimer() {
                     }
                   });
   });
+  auto arena = SimpleArenaAllocator()->MakeArena();
```

Comment:
Doesn't it seem a bit odd to create an arena for something that isn't actually a call?  Won't this wind up wasting memory on a per-channel basis?

Reply:
There's nothing call specific about arenas. I think the viewpoint at this point is that arenas are carriers for context for an activity/party. That might be a call, but it just as well could be some background async processing for a channel. The cost for this is currently a little higher than I'd like, but likely overall better than needing an additional thread-local per thread for this purpose.